### ISSUE TITLE
Cannot instatiate PptxGenJS

### ISSUE THREAD

**Issue by LucaLovagnini** [gitbrent/PptxGenJS#302]
I'm trying to use [PptxGenJS](https://gitbrent.github.io/PptxGenJS/) with webpack.

I'm using the following code:

        var pptxgenjs = require('pptxgenjs');
        var pptx = new pptxgenjs.constructor();
c        var slide = pptx.addNewSlide();
        slide.addText('Hello World!', { x:1.5, y:1.5, fontSize:18, color:'363636' });
        pptx.save('Sample Presentation');

However, I get:

    Uncaught (in promise) TypeError: fs.writeFile is not a function
        at pptxgen.js:432
        at <anonymous>

Following [this](https://github.com/gitbrent/PptxGenJS/issues/286) issue and [this](https://gitbrent.github.io/PptxGenJS/docs/integration.html) page I've set:

    node: { fs: "empty" }

and tried to call:

    pptx.setBrowser(true);

However I get:

    pptx.setBrowser is not a function

Can you please help me to solve this?

Notice that `Cannot instatiate PptxGenJS`shows correctly `1.7.0`

**Comment by gitbrent** [comment]
Hi @LucaLovagnini,

What version are you using?

None of the library examples uses `constructor()`, so that may be the issue.

The NodeJS demo (`nodejs-demo.js`) uses a different syntax:
```javascript
var PptxGenJS = require("pptxgenjs");
var pptx = new PptxGenJS();
```

**Comment by LucaLovagnini** [comment]
I was using version `1.7.0`, while I needed version `2.0.0`, since `setBrowser` is not available in `1.7.0`.

**Comment by snidell** [comment]
your NPM doc for node is out of date

var pptx = new PptxGenJS();
var slide = pptx.addNewSlide();
slide.addText('Hello World!', { x:1.5, y:1.5, fontSize:18, color:'363636' });
pptx.save('Sample Presentation');

This breaks. may want to change it to the above

var PptxGenJS = require("pptxgenjs");
var pptx = new PptxGenJS();
